### DISCUSSION TITLE
文件索引丢失，无法显示目录下的笔记文档

### DISCUSSION THREAD

**Discussion by rockshi8519** [tamlok/vnote#2061] [Q&A]
不知道操作什么了，突然只能显示目录结构和目录，无法显示目录下的笔记文档了，该如何重建索引，还是怎么导入，曾尝试重建笔记数据库，不太管用

**Comment by tamlok** [comment]
右上角，设置，重置布局看看

**Comment by rockshi8519** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    你好：已经解决了删除windows用户目录下，vnote的配置文件重新建立了一个空目录，然后新建笔记文件选择这个空目录，然后将原来的笔记文件目录，导入就重新可用了，如下：刚刚有问题时，是看不到目录下文件的
    
        
            
        
    
    
     
    
        
                    ***@***.***
        
    
 

    在2022年3月3日 15:52，Le ***@***.***> 写道： 




右上角，设置，重置布局看看
